esmvaltool/diag_scripts/kcs/resampling.py

```diff
@@ -104,11 +164,38 @@ def within_bounds(values, bounds):
     return values.between(low, high)
 
 
+def get_relative_values(top1000, info, period):
```

Comment:
```suggestion
def select_percentile_bounds(top1000, info, period):
```
I think `get_relative_values` is not very clear. My suggestions is also not great, but I can't think of anything better so far.